### ISSUE TITLE
[SF][A11Y][SAP_Horizon][DateRangePicker]: Error icons are fails to meet color contrast ratio against background color

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5595]
**Describe the bug**

There is a a11y ticket raised by QAS team about DateRangePicker component. Pls take a look.
When user navigate with keyboard on the "Start Date" button -> activate by pressing ENTER key and tab on "Set Custom Range" -> On press tab -> Type invalid date -> Error message appeared -> Error icon are fails to meet color contrast ratio of 4.5:1 against background color.

**To Reproduce**
1. Login to Instance: https://qacand.hcm.ondemand.com/login 
2. Type search 400 in the Search input field from the Opportunity Marketplace
3. Assignment Filter results page has been opened.

You can also go to this url directly after login:
 https://qacand.hcm.ondemand.com/sf/ompsearch?mock&sap-ui-theme=sap_horizon&_s.crb=EMd7AtQ%252binwRpykKhyoqOnEwyQtpvFtQMXsASCba44o%253d#/assignment?search=400

**Screenshots**
<img width="1647" alt="image" src="https://user-images.githubusercontent.com/78520316/182063799-ced7b9f3-ddac-4498-8391-cf25bff35ef3.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: “1.4.0”
`@ui5/webcomponents-react` version: “0.25.0"

**Comment by MapTo0** [comment]
Dear @sfsf-xweb-sh ,

I consulted with our Visual Design team.

The icon needs 3:1 to the background.
The 4.5:1 ratio just applies to text, not to icons.

Regards,
Martin
